.gitignore

```diff
@@ -9,4 +9,4 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-poetry.lock
+poetry.lock
```

Comment:
Can you please add back the newline?

Reply:
Sure, I didn't meant to remove it in the first place.